src/components/DocsFooter.tsx

```diff
@@ -82,7 +82,9 @@ function FooterLink({
         <span className="block no-underline text-sm tracking-wide text-secondary dark:text-secondary-dark uppercase font-bold group-focus:text-link dark:group-focus:text-link-dark group-focus:text-opacity-100">
           {type}
         </span>
-        <span className="block text-lg group-hover:underline">{title}</span>
+        <span className="block text-lg group-hover:underline break-all">
```

Comment:
I don't feel this to be elegant 
<img width="353" alt="Screenshot 2023-12-16 at 2 15 19 PM" src="https://github.com/reactjs/react.dev/assets/32865581/121b0c05-5078-43ee-9c5d-90f83cf789ef">

I'd rather consider removing the  `md:w-80` or removing explicit width and adding `md:w-auto` along with a min width so that the width takes upon content's width when it is pretty long else it will take the default min width

<img width="517" alt="Screenshot 2023-12-16 at 2 15 57 PM" src="https://github.com/reactjs/react.dev/assets/32865581/32812ff0-3fba-4548-8024-c041e8a3396b">

Reply:
Sure, here is the updated UI
<img width="691" alt="image" src="https://github.com/reactjs/react.dev/assets/99647439/e08f2bc0-6e1c-4006-a154-265ea6fb22dd">